test/non-functional/auth/my-privileges/global-roles/global-hub-admin/user.it-spec.ts

```diff
@@ -0,0 +1,76 @@
+import {
+  createUserWithParams,
+  getUser,
+  removeUser,
+} from '@test/functional-api/user-management/user.request.params';
+import { TestUser } from '@test/utils';
+import {
+  assignUserAsGlobalHubsAdmin,
+  removeUserAsGlobalHubsAdmin,
+} from '@test/utils/mutations/authorization-mutation';
+import { uniqueId } from '@test/utils/mutations/create-mutation';
+
+const userEmail = `hub${uniqueId}@alkem.io`;
+const cgrud = ['CREATE', 'GRANT', 'READ', 'UPDATE', 'DELETE'];
```

Comment:
This `cgrud` is a constant that is shared in different files, extract in a a common shared location and re-use it.